<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayClass</class>
 <widget class="QWidget" name="PlayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Question">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>125</y>
     <width>1200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AnswerA">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>300</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnswerB">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>300</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnswerC">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>300</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnswerD">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>300</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
